tutorials/intermediate/02-Machine_Translation_using_PyTorch_Ignite.ipynb

```diff
@@ -722,7 +722,7 @@
     "\n",
     "\n",
     "def get_save_handler(config):\n",
-    "    return DiskSaver(config[\"output_path_\"], require_empty=False)"
+    "    return config[\"output_path_\"]"
```

Comment:
Hi @anirudhb11 thanks for the changes! Since now it is returning only a path (and not a handler) should we change the name of the function to something like `get_save_path` ?

Reply:
I thought over this, you're right that it returns the path, but if we do this we would have code like: `save_handler=get_save_path(config)` in some places which might also not be good. Something that would look better would be to modify it to `save_path=get_save_path(config)` and then we also change the `Checkpoint` class to take in `save_path` instead of `save_handler`. Let me know your thoughts.